rasa/utils/tensorflow/rasa_layers.py

```diff
@@ -678,20 +678,24 @@ def call(
 
 
 class RasaSequenceLayer(RasaCustomLayer):
-    """Creates an embedding from all features for a sequence attribute; facilitates MLM.
-
-    This layer combines all features for an attribute and embeds them using a
-    transformer, optionally doing masked language modeling. The layer is meant only for
-    attributes with sequence-level features, such as `text`, `response` and
-    `action_text`.
-
-    Internally, this layer applies the following steps:
-    1. Combine features using `RasaFeatureCombiningLayer`.
-    2. Apply a dense layer(s) to the combined features.
+    """A sequence-to-sequence model for embeddings that optionally creates MLM examples.
```

Comment:
According to search: MLM = Multi-Level-Marketing? 😂

Reply:
Yes, it is something used to make the Sesame Street more popular :D

Reply:
I mean we shouldn't use abbreviations in doc strings. I see that "masked language model" would be too long to fit in the tagline, but it could go in as a sentence below.